.stylelintrc

```diff
@@ -0,0 +1,109 @@
+{
+  "rules": {
```

Comment:
Is there a reason why we don't extend the standard: https://github.com/stylelint/stylelint-config-standard

```sh
npm install stylelint-config-standard --save-dev
```

```js
{
  "extends": "stylelint-config-standard",
  "rules": {
    ...
  }
}
```